provider/pkg/await/awaiters.go

```diff
@@ -51,7 +51,6 @@ type createAwaitConfig struct {
 	initialAPIVersion string
 	logger            *logging.DedupLogger
 	clientSet         *clients.DynamicClientSet
-	currentInputs     *unstructured.Unstructured
```

Comment:
I see no reason for the awaiters to have or to use the inputs, since they're used strictly after any desired changes have been applied.